.github/workflows/release_linux_aarch64.yml

```diff
@@ -0,0 +1,137 @@
+name: LinuxRelease_aarch64
+
+on:
+  schedule:
+    # Run weekly on Monday 00:00
+    - cron:  '00 00 * * MON'
+  push:
+    branches: [master, rel-*]
+  pull_request:
+    branches: [rel-*, master]  # TODO remove master before merge
+  workflow_dispatch:
+
+jobs:
+  build:
+    if: github.event_name != 'pull_request' || startsWith( github.base_ref, 'rel-') || contains( github.event.pull_request.labels.*.name, 'run release CIs')
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # the different python versions for building wheels
+        python-version: [cp37-cp37m, cp38-cp38, cp39-cp39]
+    env:
+      # setting up python and docker image
+      py: /opt/python/${{ matrix.python-version }}/bin/python
+      img: quay.io/pypa/manylinux2014_aarch64
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Checkout submodules
+      shell: bash
+      run: |
+        auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+        git submodule sync --recursive
+        git -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive --depth=1
+
+      # setting up qemu for enabling aarch64 binary execution on x86 machine
+    - uses: docker/setup-qemu-action@v1
+
+      # Creating a virtual environment on machine with the help of docker container \
+      # and installing the dependencies inside that \
+      # so that we can use installed dependencies.
+    - name: Install dependencies
+      run: |
+        docker run --rm -v ${{ github.workspace }}:/ws:rw --workdir=/ws \
+          ${{ env.img }} \
+          bash -exc '${{ env.py }} -m pip install virtualenv && ${{ env.py }} -m venv .env && \
+          source .env/bin/activate && \
+          ${{ env.py }} -m pip install -U numpy protobuf==3.11.3 && \
+          yum install -y protobuf-compiler protobuf-devel
```

Comment:
I assume the protobuf version from yum package is not 3.11.3, right? I think a better way is to install protobuf from source like this: https://github.com/onnx/onnx/blob/31f0fd6103e231664d1cfd3b25264f672f5de960/.github/workflows/manylinux/entrypoint.sh#L18

Reply:
Yes, I checked it, it is not required as it is already installed.